src/app/gene-plot/gene-plot.component.ts

```diff
@@ -345,6 +346,8 @@ export class GenePlotComponent implements OnChanges {
 
   private drawVariants(): void {
     const variantsElement = this.plotElement.append('g').attr('id', 'variants');
+    let c = 0
```

Comment:
Avoid nondescript names for variables. Use `counter`, `somethingCounter` or `numberOfSomething`